MdePkg/Test/Library/RngLibHostTestLfsr/RngLibHostTestLfsr.c

```diff
@@ -0,0 +1,124 @@
+/** @file -- RngLibHostTestLfsr.c
+A minimal implementation of RngLib that supports host based testing
+with a simple LFSR:
+https://en.wikipedia.org/wiki/Linear-feedback_shift_register
+
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Library/RngLib.h>
+
+STATIC
+UINT16
+LfsrXorshift16 (
+  VOID
+  )
+{
+  static UINT16  Lfsr = 0xACE1;         /* Any nonzero start state will work. */
+
+  // 7,9,13 triplet from http://www.retroprogramming.com/2017/07/xorshift-pseudorandom-numbers-in-z80.html
+  Lfsr ^= Lfsr >> 7;
+  Lfsr ^= Lfsr << 9;
+  Lfsr ^= Lfsr >> 13;
+
+  return Lfsr;
+}
+
+/**
+  Generates a 16-bit random number.
+
+  if Rand is NULL, then ASSERT().

```

Comment:
This behavior isn't actually in the function.

Reply:
Thanks. I'll update them.